samples/boards/st/mco/sample.yaml

```diff
@@ -2,5 +2,6 @@ sample:
   name: STM32 microcontroller clock output (MCO) example
 tests:
   sample.board.stm32.mco:
-    platform_allow: nucleo_u5a5zj_q stm32f746g_disco
+    platform_allow: nucleo_u5a5zj_q stm32f746g_disco nucleo_f411re
+    filter: not CONFIG_SOC_SERIES_STM32WB0X
```

Comment:
Why specifically WB0 ? There are other series not enabling MCO.
Also, filter should not be required if `platform_allow` is used